rules.d/70-uaccess.rules.in

```diff
@@ -99,4 +99,14 @@ SUBSYSTEM=="hidraw", ENV{ID_AV_PRODUCTION_CONTROLLER}=="1", TAG+="uaccess"
 # This also allows accessing HID devices with the libusb backend of hidapi.
 SUBSYSTEM=="usb", ENV{ID_AV_PRODUCTION_CONTROLLER}=="1", TAG+="uaccess"
 
+# Trezor HW wallets
+# Trezor
+SUBSYSTEM=="usb", ATTR{idVendor}=="534c", ATTR{idProduct}=="0001", TAG+="uaccess"
```

Comment:
Please don't hardcode devid in udev rules, but assign a "category" for them first and use it here (refer to `ID_AV_PRODUCTION_CONTROLLER` above as an example)

BTW, is this actually safe? Note that there's an proposal to add support for revoking hidraw devices in kernel and logind: https://github.com/systemd/systemd/pull/29797.

Reply:
Thanks, I'll redo it in a similar way as in the PR that introduced `ID_AV_PRODUCTION_CONTROLLER`

Do you have any preference regarding the hwdb file name? Something like `hw-wallets.hwdb` where I would attach `TREZOR_HW_WALLET=1` to the trezor models?
Should I create a new one or use the `ID_MODEL_FROM_DATABASE` as described further down?

I found that trezor devices are already listed and have `ID_MODEL_FROM_DATABASE.` Can I match based on that so I don't introduce a new file as it was for the AV devices?

```
usb:v534C*
 ID_VENDOR_FROM_DATABASE=SatoshiLabs
usb:v534Cp0001*
 ID_MODEL_FROM_DATABASE=Bitcoin Wallet [TREZOR]
usb:v534Cp0002*
 ID_MODEL_FROM_DATABASE=Bitcoin Wallet [TREZOR v2]
usb:v1209p53C1*
 ID_MODEL_FROM_DATABASE=SatoshiLabs TREZOR
usb:v1209p53C0*
 ID_MODEL_FROM_DATABASE=SatoshiLabs TREZOR Bootloader
```
But in that case, the udev rules would look a bit messy because of the long model names that need to be matched.

Reply:
I have added the fixup commit with the first option, and it's pretty much the same as what was done for the AV devices. I tested it locally on my Linux desktop, and the `trezord` recognized all of the devices correctly.